frontend/src/components/v1/RelatedRecordActionRow.css

```
@@ -0,0 +1,193 @@
+}
+
+.related-record-action-row__btn:disabled {
+  opacity: 0.5;
+  cursor: not-allowed;
+}
+
+.related-record-action-row__spinner {
+  width: 12px;
+  height: 12px;
+  border: 2px solid rgba(255, 255, 255, 0.3);
+  border-top-color: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)